Updated manuscript: Add IHG, ATRT, and neuroblastoma subtyping (#19)
d3b-center/OpenPedCan-methods: content/06.methods.md

In OpenPedCan, the molecular subtyping of ATRT was based solely on the DNA methylation data. 
Briefly, ATRT samples with a high confidence DKFZ methylation subclass score (>= 0.8) were selected and subtypes were assigned based on the DKFZ methylation subclass [@doi10.1038/nature26000]. 
Samples with low confidence DKFZ methylation subclass scores (< 0.8) were identified as "ATRT, To be classified". 

**Neuroblastoma tumors.**

Neuroblastoma (NBL) tumors with a pathology diagnosis of neuroblastoma, ganglioneuroblastoma, or ganglioneuroma were subtyped based on their MYCN copy number status as either "NBL, MYCN amplified" or "NBL, MYCN non-amplified". 
If `pathology_free_text_diagnosis` was "NBL, MYCN non-amplified" and the genetic data suggested MYCN amplification, the samples were subtyped as "NBL, MYCN amplified". 
On the other hand, if `pathology_free_text_diagnosis` was "NBL, MYCN amplified" and the genetic data suggested MYCN non-amplification, the RNA-Seq gene expression level of _MYCN_ was used as a prediction indicator. 

> In those cases, samples with _MYCN_ gene expression above or below the cutoff (TPM >= 140.83) were subtyped as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 

Reviewer: @adilahiri @afarrel can you comment how 140.83 was chosen?

Reviewer 2: @jharenza: The value was established as a ball park estimate by looking at one of the TPM plots produced in the module. It was not a calculated value.  
If you see case 4 in the documentation of the[ molecular-subtyping-NBL](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-NBL) that is what it explains. 

I looked at the image and the code there was a sample "TARGET-30-PAMEZH-01A-01R" marked using a gray bar just below the cutoff TPM value of 140.83 (marked by the black line). This sample had highest TPM below the cut off in the image. As I remember with my discussions with @ewafula we came to the conclusion that the drop in TPM value for this sample from the sample with the next largest TMP value was significant "TARGET-30-PAPKXS-01A-01D" (green bar) compared to the next two pairs of samples with higher TPM values.

Reviewer: Ah, I see - Thanks for the explanation. Looking at it, I think we might have been able to go lower, as I see a lot of amp samples below that cutoff. We may want to plot by clinical MYCN status (FISH) just to see what the cutoff looks like then), but obviously beyond the scope here. Will just reword this a bit. Thanks.

> In those cases, samples with _MYCN_ gene expression above or below the cutoff (TPM >= 140.83) were subtyped as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 

Reviewer: ```suggestion
In those cases, samples with _MYCN_ gene expression above or below the cutoff (TPM >= 140.83 based on visual inspection of MYCN CNV status) were subtyped as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 
```

In OpenPedCan project, the molecular subtyping of ATRT is based on the DNA methylation data. 
Briefly, samples with "Atypical teratoid rhabdoid tumors" pathology diagnosis would be considered as ATRT. 
To molecularly subtype ATRT, samples with high DKFZ methylation subclass score (>= 0.8) would be selected and the subtyping will be assigned based on the DKFZ methylation subclass ([@https://doi.org/10.1038/nature26000]). 
Samples with poor DKFZ methylation subclass score (< 0.8) would be identified as "ATRT, To be classified". 

Neuroblastoma tumors...

Neuroblastoma (NBL) is a type of cancer derived from nerve tissue and usually observed in the kids under age 5. 
In OpenPedCan project, NBL contains the samples with pathology diagnosis of neuroblastoma, ganglioneuroblastoma, and ganglioneuroma. 
These samples can be further subtyped based on the status of MYCN amplification. 
Combining the clinical pathology diagnosis and copy-number variance (CNV) calling from DNA seqeuncing data, samples can be subtyped into "NBL, MYCN amplified" or "NBL, MYCN non-amplified". 

> If free-text pathology diagnosis is "NBL, MYCN non-amplified" while the genetic data suggested MYCN amplification, the samples would be assigned as "NBL, MYCN amplified". 
> On the other hand, if free-text pathology diagnosis was "NBL, MYCN amplified" while the genetic data suggested MYCN non-amplification, gene expression level of MYCN from RNA-seq would be used as a prediction indicator. 
> Samples with MYCN gene expression above/below the cutoff (TPM >= 140.83) would be identified as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 
> Samples without DNA sequencing data would also use MYCN gene expression for molecular subtyping. 
> If a sample does not fit none of these situations, this sample would be classified as "NBL, To be classified".  

Reviewer: ```suggestion
If `pathology_free_text_diagnosis` was "NBL, MYCN non-amplified" and the genetic data suggested MYCN amplification, the samples were subtyped as "NBL, MYCN amplified". 
On the other hand, if `pathology_free_text_diagnosis` was "NBL, MYCN amplified" and the genetic data suggested MYCN non-amplification, the RNA-Seq gene expression level of _MYCN_ was used as a prediction indicator. 
In those cases, samples with _MYCN_ gene expression above or below the cutoff (TPM >= 140.83) were subtyped as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 
_MYCN_ gene expression was also used to subtype samples without DNA sequencing data. 
If a sample did not fit none of these situations, it was denoted as "NBL, To be classified".  
```

(TPM >= 140.83) - can you say more about how this was chosen?
